src/pages/ReimbursementAccount/BankInfo/substeps/Manual.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {useCallback} from 'react';
+import {Text} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx/lib/types';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import TextInput from '@components/TextInput';
+import useLocalize from '@hooks/useLocalize';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import ExampleCheckImage from '@pages/ReimbursementAccount/ExampleCheck';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReimbursementAccount} from '@src/types/onyx';
+
+type ManualOnyxProps = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+};
+
+type ManualProps = ManualOnyxProps & SubStepProps;
+
+type FormValues = {routingNumber: string; accountNumber: string};
+
+const bankInfoStepKeys = CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY;
+
+function Manual({reimbursementAccount, onNext}: ManualProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const defaultValues: FormValues = {
+        [bankInfoStepKeys.ROUTING_NUMBER]: reimbursementAccount?.achData?.[bankInfoStepKeys.ROUTING_NUMBER] ?? '',
+        [bankInfoStepKeys.ACCOUNT_NUMBER]: reimbursementAccount?.achData?.[bankInfoStepKeys.ACCOUNT_NUMBER] ?? '',
+    };
+
+    /**
+     * @param {Object} values - form input values passed by the Form component
+     * @returns {Object}
+     */
```

Comment:
Let's remove jsdoc without description in all files + remove types from jsdoc, can someone check why it's not removed automatically here?
```suggestion
    /** @param values - form input values passed by the Form component */
```